source/applications/advanced/hand_eye_calibration/ur_hand_eye_calibration/universal_robots_perform_hand_eye_calibration.py

```diff
@@ -408,7 +408,7 @@ def perform_hand_eye_calibration(mode: str, data_dir: Path):
                 raise RuntimeError(f"Failed to detect feature points from frame {frame_file}")
 
             print(f"Read robot pose from pos{idata:02d}.yaml")
-            with open(pose_file) as file:
+            with open(pose_file, encoding="utf-8") as file:
```

Comment:
did you consider Path(pose_file).read_text(). I know that there are issues with encoding there as well..

Reply:
No, I haven't. This one is embedded so I prefer it for that reason. I think it is nice to have consistent between samples and some of them will not import Path.